### PR TITLE
Add unit tests

### PR DESCRIPTION
**Pull request by kseniadumpling**
This PR

- Adds unit & integration tests for components and containers
- Adds testing modules from npm (updates package.json & package-lock.json)
- Adds some id attributes to the tags
- Fixes some inconsistencies

### CODE REVIEW

**Comment by egorlitsky** [comment]
Will review ;)

File: package.json [hunk 1]
```diff
@@ -25,10 +26,11 @@
     "babel-preset-react": "^6.11.1",
     "babel-register": "^6.9.0",
     "bonjour": "3.5.0",
-    "bootstrap": "^4.0.0-beta",
+    "bootstrap": "^4.0.0",
+    "chai": "^4.2.0",
```
Comment (egorlitsky) [inline package.json]: The same comment as for TestCafe change

File: src/app/containers/HelpPage.js [hunk 2]
```diff
@@ -57,7 +57,7 @@ class HelpPage extends React.Component {
     }
 
     render() {
-        if (this.state.state == "expect") {
+        if (this.state.state === "expect") {
```
Comment (egorlitsky) [inline src/app/containers/HelpPage.js]: `state` with `state` identifier? :trollface:

File: src/test/containers/AppliancePageTest.js [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+        );
+        wrapper.find("#pageButtonConfigured").simulate("click");
+        wrapper.update();
+
+        let page = wrapper.find(AppliancePage);
+        const firstCluster = page.find(Appliance).at(0);
+        const secondCluster = page.find(Appliance).at(1);
+
+        const clusters = testData.storages.filter(appliance => {
+            return appliance.cluster === "true";
```
Comment (egorlitsky) [inline src/test/containers/AppliancePageTest.js]: What's the reason `boolean` inside the `string`?

Comment (kseniadumpling) [reply src/test/containers/AppliancePageTest.js]: Actually, it's a problem not only with a test, but also with the way how we store data. 
Will fix in next PR

Comment (egorlitsky) [reply src/test/containers/AppliancePageTest.js]: OK

**Pull request merged**